src/spotify-card.ts

```diff
@@ -293,21 +294,32 @@ export class SpotifyCard extends LitElement {
   }
 
   private onShuffleSelect(): void {
-    if (this._spotify_state?.state == 'playing') {

```

Comment:
Why have you done this check? It works even when pasued

Reply:
Right. Whqt happens if nothing is playing or is it hidden then?

Reply:
Nothing happens. No, it's always visible, I don't like it when the controls shift through the UI

Reply:
Maybe we can hide the controls when no device is selected

Reply:
You can choose. I dont have an opinion really